addons/web_editor/static/src/js/editor/odoo-editor/src/commands/deleteForward.js

```diff
@@ -29,7 +31,7 @@ Text.prototype.oDeleteForward = function (offset, alreadyMoved = false) {
     if (offset === this.nodeValue.length) {
         // Delete at the end of a text node is not a specific case to
         // handle, let the element implementation handle it.
-        HTMLElement.prototype.oDeleteForward.call(this, offset, alreadyMoved);
+        HTMLElement.prototype.oDeleteForward.call(parentNode, [...parentNode.childNodes].indexOf(this) + 1);
```

Comment:
This is quite the change here. If there was a bug with this function then it would be better to have a specific commit with this change and a particular test that breaks with the current code but doesn't with your new one.

Reply:
It is not realy a bug but almost, let me explain.

When we tgigger the `HTMLElement.prototype.oDeleteForward` for now we give it a node (this), so in some case the `this` in **HTMLElement** might not be a HTMLElement. which is wrong I thing.

The fact is that all the trial and error we do in those oDelete function make it not a problem because the system is fail safe enough. but I change it because I encountered a problem in the code I added if I was not able to expect an HTMLElement.

Functionally is should not change anything, and all the editor Unit tests still pass.

Obviously I can put this in his own commit.

Reply:
We should discuss this this afternoon.